support-workers/src/test/scala/com/gu/support/workers/JsonFixtures.scala

```diff
@@ -64,7 +65,7 @@ object JsonFixtures {
           "isTestUser": false
         }
     """
-  val requestIdJson = "\"requestId\": \"e18f6418-45f2-11e7-8bfa-8faac2182601\""
+  def requestIdJson: String = s""""requestId": "${UUID.randomUUID()}\""""
```

Comment:
this is randomised so that the end to end test will follow the subscribe path rather than the skipSubscribe one.  randomness in tests is a mixed thing, generally to be avoided in predictable tests but I think it's ok in this case.  We're not randomising the test behaviour, just an opaque field.